src/custom_types.d.ts

```diff
@@ -0,0 +1,55 @@
+interface BinaryWriterStatic {
+	new(): BinaryWriter;
+}
+
+interface BinaryWriter {
+	writeString(): void;
+	writeUint64(): void;
+}
+
+interface BinaryReaderStatic {
+	new(bytes: Uint8Array): BinaryReader;
+}
+
+interface BinaryReader {
+	readString(): string;
+	readUint64(): number;
+	isEndGroup(): boolean;
+	nextField(): void;
+	getFieldNumber(): number;
+	skipField(): void;
+}
+
+interface Uint64 {
```

Comment:
the additional types which we can use later are stored in this [gist](https://gist.github.com/subash-a/62397f16b3a05fdf625a)